src/screens/CreateWalletScreen.tsx

```diff
@@ -222,60 +216,85 @@ export class CreateWalletScreen extends React.PureComponent<Props, State> {
     return (
       <>
         <Text style={styles.advancedOptionsLabel}>{i18n.wallets.add.walletType}</Text>
+
         {!isAdvancedOptionsEnabled ? (
-          <RadioGroup color={palette.secondary} onSelect={this.onSelect} selectedIndex={this.state.selectedIndex}>
-            <RadioButton style={styles.radioButton} value={HDSegwitP2SHArWallet.type}>
-              <View style={styles.radioButtonContent}>
-                <Text style={styles.radioButtonTitle}>{HDSegwitP2SHArWallet.typeReadable}</Text>
-                <Text style={styles.radioButtonSubtitle}>{i18n.wallets.add.ar}</Text>
-              </View>
-            </RadioButton>
-            <RadioButton style={styles.radioButton} value={HDSegwitP2SHAirWallet.type}>
-              <View style={styles.radioButtonContent}>
-                <Text style={styles.radioButtonTitle}>{HDSegwitP2SHAirWallet.typeReadable}</Text>
-                <Text style={styles.radioButtonSubtitle}>{i18n.wallets.add.air}</Text>
-              </View>
-            </RadioButton>
-            <RadioButton style={styles.radioButton} value={HDSegwitP2SHWallet.type}>
-              <View style={styles.radioButtonContent}>
-                <Text style={styles.radioButtonTitle}>{i18n.wallets.add.legacyTitle}</Text>
-                <Text style={styles.radioButtonSubtitle}>{i18n.wallets.add.legacy}</Text>
-              </View>
-            </RadioButton>
-          </RadioGroup>
+          <>
+            <RadioButton
```

Comment:
https://ant.design/components/radio/ check how the Radio is component is constructed here. It would be more flexible if the component will accept the `checked` props. 
Proposal:
 <RadioButton
              title={HDSegwitP2SHAirWallet.typeReadable}
              subtitle={i18n.wallets.add.air}
              value={HDSegwitP2SHAirWallet}
              checked={this.state.walletClass.type === HDSegwitP2SHAirWallet.type}
              onPress={value => {
                this.setState({ walletClass: value });
              }}
            />

Reply:
I like it. Done.